### ISSUE TITLE
Pagination is not accessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6890]
## Environment

> Operating system

Windows 10 latest

> Browser

Edge, Firefox

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description
Pagination has the following errors in WIndows High Contrast Mode
- the forward arrow is too low contrast
- the down arrows are too low contrast
- on Firefox there are no focus states

Firefox
![image](https://user-images.githubusercontent.com/40970507/93942679-83b43080-fcf6-11ea-90ce-aefac9bc2193.png)

Edge
![image](https://user-images.githubusercontent.com/40970507/93942692-8a42a800-fcf6-11ea-8395-161807b39faa.png)
